### PR TITLE
Strategies balances indexing

### PR DESCRIPTION
**Pull request by rcstanciu**
- Created an abstraction class `Strategy` for fetching the balance of each strategy
- Added a function running once per day for indexing the balance of each strategy
- Created an API `GET /strategies` for fetching the latest balance of each strategy
- Logged APY aberrations and ignored negative APYs

### CODE REVIEW

File: indexer.py [hunk 1]
```diff
@@ -79,6 +81,7 @@ def __init__(self, blocks_per_call=None):
             self.index_apy: settings.INDEXER_STATS_BLOCKS_PER_CALL,
             # 268 blocks is roughly every hour on current Ethereum mainnet
             self.reset_apy_calc: 268,
+            self.index_strategy_balances: 1,
```
Comment (Evert0x) [inline indexer.py:84]: Every block? :astonished:

File: indexer.py [hunk 2]
```diff
@@ -129,11 +132,24 @@ def calc_factors(self, session, indx, block):
...
+        if apy < 0:
+            logger.warning("APY %s is being skipped because is negative." % apy)
+            return
+
+        if apy > 0.15:
+            logger.warning("APY %s is being skipped because is higher than 15%%." % apy)
+            return
+
+        if indx.apy != 0 and apy > indx.apy * 2:
+            logger.warning("APY %s is 2 times higher than the previous APY of %s" % (apy, indx.apy))
```
Comment (Evert0x) [inline indexer.py:150]: Can be removed as this is related to non incrementing APYs

File: strategies/strategies.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            return None
+        except Exception as e:
+            logger.exception(e)
+            return None
+
+
+class Strategies:
+    AAVE = Strategy(address="0xE3C37e951F1404b162DFA71A13F0c99c9798Db82", name="Aave")
+    COMPOUND = Strategy(address="0x8AEA96da625791103a29a16C06c5cfC8B25f6832", name="Compound")
+    EULER = Strategy(address="0x9a902e8Aae5f1aB423c7aFB29C0Af50e0d3Fea7e", name="Euler")
```
Comment (Evert0x) [inline strategies/strategies.py:63]: Check docs PR for correct addresses

File: indexer.py (hunk 1 above)
Comment (rcstanciu) [reply indexer.py:84]: Used for testing locally, forgot to change back. Thanks!

**Review by Evert0x — APPROVED** [review]

**Pull request merged**